### DISCUSSION TITLE
Server Error Error: Response not successful: Received status code 400

### DISCUSSION THREAD

**Discussion by dhawansolanki** [RakeshPotnuru/Blog#8] [Q&A]
![Error2](https://user-images.githubusercontent.com/91565429/185114777-6c54b263-a571-4bfe-81b2-bf683c97e9a6.JPG)

Not able to rectify and solve the error (attached a screenshot of error)..

The Error Appears when we view the article...
Please Help...

**Comment by RakeshPotnuru** [comment]
@dhawansolanki 
Have you included these environment variables in the `.env.local` file?
```bash
NEXT_PUBLIC_HYGRAPH_CONTENT_API_URL=
NEXT_PUBLIC_HYGRAPH_AUTH_TOKEN=
```

**Reply by dhawansolanki** [reply]
Yes...
I can Upload posts and that is visible on landing page..
But when click on article to read more .. the above error appears 
Can you refer to my updated code at 
https://github.com/dhawansolanki/B
